test/parallel/test-openssl-ca-options.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+// This test checks the usage of --use-bundled-ca and --use-openssl-ca arguments
+// to verify that both are not used at the same time.
+const common = require('../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+const assert = require('assert');
+const childProcess = require('child_process');
+const result = childProcess.spawnSync(process.execPath,
+                                      ['--use-bundled-ca', '--use-openssl-ca'],
+                                      {encoding: 'utf8'});
+
+assert.strictEqual(result.stderr,
+                   process.execPath + ': either --use-openssl-ca or ' +
+                   '--use-bundled-ca can be used, not both\n');
+assert.strictEqual(result.status, 9);
```

Comment:
for completeness, this should likely launch two other processes, each respectively using only `--use-bundled-ca` and `--use-openssl-ca` to make sure that the error is not emitted in those cases.